### DISCUSSION TITLE
Agentless Capabilities

### DISCUSSION THREAD

**Discussion by ljtill** [community/community#17021] [Actions]
**Feature**

Similar to Azure Pipelines, it would be useful to provide the capability to invoke agentless tasks within GitHub Actions workflows.

**Docs**

[Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#agentless-tasks)

**Comment by Len4i** [comment]
Indeed, especially for things like [handling-skipped-but-required-checks](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks) where it is an official workaround to create dummy action

**Comment by Fitzbert-Fitz** [comment]
Comming from azure devops, I am a little surprised that this does not exist.
